modules/ROOT/pages/_partials/attributes.adoc

```diff
@@ -5,32 +5,26 @@
 :service-name:  
 
 :mobile-client: mobile app record
-
 :mobile-cli: Mobile CLI
 
-:metrics-service: Metrics
 :grafana-ui: Grafana
 :prometheus-ui: Prometheus
     
 :keycloak-service: Identity Management
 :keycloak-ui: Keycloak admin UI
 :keycloak-dashboard: Auth Dashboard
     
-:unifiedpush-service: Push
 :push-ui: Unified Push admin UI
-   
-:digger-service: CI\CD
+:push-notification: push notification
+:push-notifications: push notifications
```

Comment:
@pb82 @finp 
Is this correct? Having constants like this? (singular/plural)

Reply:
No, i don't think this works unless both are defined. Do this instead: `:push-notifications:s`